src/Microsoft.Health.Dicom.Core/Modules/ServiceModule.cs

```diff
@@ -166,7 +166,12 @@ public void Load(IServiceCollection services)
                    .AsSelf()
                    .AsImplementedInterfaces();
 
-                services.Add<AddExtendedQueryTagService>()
```

Comment:
Did you remove `AddExtendedQueryTagService` or is the diff misleading?

Reply:
oops, good catch